charts/frontend/Chart.yaml

```diff
@@ -15,6 +15,10 @@ dependencies:
   version: 6.17.x
   repository: https://charts.bitnami.com/bitnami
   condition: rabbitmq.enabled
+- name: mongodb
+  version: 10.26.4
```

Comment:
Set the patch version to "x" so helm dependencies can upgrade to the latest patch version